mass/core/files.py

```diff
@@ -251,14 +251,18 @@ def _parse_header(self):
         # Read the new (Feb 2024) subframe information. If missing, assume the old TDM values pertain
         # (so # of rows -> subframe divisions, and row # -> subframe offset), unless source is Abaco,
         # in which case use 64 subframe divisions and offset of 0.
-        default_divisions = self.number_of_rows
         default_offset = self.row_number
+        default_divisions = self.number_of_rows
+        if default_divisions <= 0:
+            default_divisions = 1
         if "Abaco" in self.source:
             # The external trigger file can override this, but assume 64 divisions at first.
             default_divisions = 64
             default_offset = 0
         self.subframe_divisions = int(header_dict.get(b"Subframe divisions", default_divisions))
         self.subframe_offset = int(header_dict.get(b"Subframe offset", default_offset))
+        if self.subframe_divisions <= 0:
```

Comment:
This is a redundant fix, if the above fix works, this isn't necessary. So future readers will have a hard time understanding why this exists and it will make reading the code harder without adding any value.

Reply:
The issue title says that the issue happane when no number of rows is stored in the LJH file. But [we have a line ](https://github.com/usnistgov/mass/blob/e4899d1d921237c142f35a3e85b4e82e04b53193/mass/core/files.py#L248) should make `number_of_rows=1` in that case, which should prevent the divide by zero. So I also don't understand from this how the issue ever happens.